Inspectors/Inspect-ExternalForwarding.json

```diff
@@ -0,0 +1,17 @@
+{
+	"FindingName": "Outbound Forwarding Enabled at Tenant!",
+	"Description": "Email forwarding can be a useful, but can also pose a security risk due to the potential disclosure of information. Attackers might use this information to attack your organization or partners. Outbound spam filter policies can be configured to block forwarding to external parties at the Tenant (global) level.",
```

Comment:
Should be "Email forwarding can be useful"

Reply:
Fixed